src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -99,28 +99,28 @@
     "ng-block-ui": "3.0.2",
     "ng-click-outside": "7.0.0",
     "ng2-charts": "2.4.2",
-    "ngx-pipe-function": "^1.0.0",
+    "ngx-pipe-function": "1.0.0",
     "ngx-toastr": "13.1.0",
     "rxjs": "6.6.3",
     "simplebar-angular": "2.3.0",
-    "swagger-ui": "3.49.0",
-    "tslib": "^2.0.0",
-    "zone.js": "0.10.3"
+    "swagger-ui": "3.52.5",
+    "tslib": "2.3.1",
+    "zone.js": "~0.11.4"
```

Comment:
```suggestion
    "zone.js": "0.11.4"
```

Reply:
This seems to be from running the `ng update`. I'll make the change, thanks